src/model/fixtures/project_state.json

```diff
@@ -0,0 +1,719 @@
+      {
+         "branchName": "branch-1",
+         "componentId": "ex-generic-v2",
+         "name": "empty",
+         "configuration": {},
+         "rows": null
+      },
+      {
+         "branchName": "branch-1",
+         "componentId": "keboola.ex-db-mysql",
+         "name": "only-connection",
+         "configuration": {
+            "parameters": {
+               "db": {
+                  "#password": "my-secret",
+                  "database": "keboola_testing",
+                  "host": "testing-mysql-wr-ex.cvkpp0ld53zd.us-east-1.rds.amazonaws.com",
+                  "networkCompression": true,
+                  "port": 3306,
+                  "ssh": {
+                     "sshPort": 22
+                  },
+                  "transactionIsolationLevel": "READ UNCOMMITTED",
+                  "user": "keboola"
+               }
+            }
+         },
+         "rows": null
+      },
+      {
+         "branchName": "branch-1",
+         "componentId": "keboola.python-transformation-v2",
+         "name": "python-transformation",
+         "configuration": {
+            "parameters": {
+               "blocks": [
+                  {
+                     "codes": [
+                        {
+                           "name": "Code X",
+                           "script": [
+                              "print(100)"
+                           ]
+                        },
+                        {
+                           "name": "Code Y",
+                           "script": [
+                              "print(200)"
+                           ]
+                        }
+                     ],
+                     "name": "Block 1"
+                  },
+                  {
+                     "codes": [
+                        {
+                           "name": "Code Z",
+                           "script": [
+                              "# Sum of natural numbers up to num\n\nnum = 16\n\nif num \u003c 0:\n   print(\"Enter a positive number\")\nelse:\n   sum = 0\n   # use while loop to iterate until zero\n   while(num \u003e 0):\n       sum += num\n       num -= 1\n   print(\"The sum is\", sum)"
+                           ]
+                        }
+                     ],
+                     "name": "Block 2"
+                  }
+               ],
+               "packages": [
+                  "xgboost"
+               ]
+            }
+         },
+         "rows": null
+      },
+      {
+         "branchName": "branch-1",
+         "componentId": "keboola.ex-db-mysql",
+         "name": "empty",
+         "configuration": {},
+         "rows": null
+      },
+      {
+         "branchName": "Main",
+         "componentId": "ex-generic-v2",
+         "name": "empty",
+         "configuration": {},
+         "rows": null
+      },
+      {
+         "branchName": "Main",
+         "componentId": "keboola.ex-azure-cost-management-sp",
+         "name": "test-schema-validation",
```

Comment:
konfiguracia so config a rows schemou (to to nevidno, ze komponenta ma schemu, no API ju vracia a uplatni sa to pri prikaze `validate`).